libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/compound/LightDesignTokens.kt

```diff
@@ -0,0 +1,333 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.libraries.designsystem.theme.compound
+
+import androidx.compose.ui.graphics.Color
+
+object LightDesignTokens {
```

Comment:
Consider making these objects `internal` so that developers don't use them.

To better enforce this separation I'd extract a `theme` module from the `designsystem` module and move all theme-related code in there.
This way we are sure that everything marked as `internal` in the `theme` module can't be used by the rest of the code.
`designsystem` can import the `theme` module as  an `api` depdendency so that we don't have to import it everywhere but we'll get it as part of the `designsystem` dependency.